resources/version.go

```diff
@@ -48,6 +51,55 @@ func CreateVersion(db *gorm.DB, version *Version) (bool, error) {
 	return result.RowsAffected == 1, result.Error
 }
 
+// versionDeploymentDefinition builds a deployment spec for the provided version
+func versionDeploymentDefinition(version *Version) *v1beta1.Deployment {
+	replicas := int32(version.Replicas)
+	name := fmt.Sprintf(
+		"%v-%v-%v",
+		version.Environment.Application.Slug,
+		version.Environment.Slug,
+		version.Slug,
+	)
+
+	return &v1beta1.Deployment{
+		ObjectMeta: v1.ObjectMeta{
+			Name:      name,
+			Namespace: "brizo",
+			Labels: map[string]string{
+				"brizoManaged": "true",
```

Comment:
@jlaswell currently we hard go the version in main.go, I think this change is a good idea but I feel like it might be out of the scope of this current goal. We're early enough in that we could go back and refactor since this is the first and only k8s resource we're creating up to this point.